pom.xml

```diff
@@ -71,7 +71,7 @@
         <netty.version>4.1.6.Final</netty.version>
         <slf4j.version>1.7.12</slf4j.version>
         <!-- If compiling with different hadoop version also modify default hadoop coordinates in TaskConfig.java -->
-        <hadoop.compile.version>2.3.0</hadoop.compile.version>
```

Comment:
@dclim 2.8 is released if that works for you i can update the dependency, please let me know.